code/modules/mob/living/silicon/ai/vox_sounds.dm

```diff
@@ -17,22 +16,23 @@ GLOBAL_LIST_INIT(vox_sounds, list("," = 'sound/vox_fem/,.ogg',
 "across" = 'sound/vox_fem/across.ogg',
 "activate" = 'sound/vox_fem/activate.ogg',
 "activated" = 'sound/vox_fem/activated.ogg',
+"active" = 'sound/vox_fem/active.ogg',
 "activity" = 'sound/vox_fem/activity.ogg',
-"adios" = 'sound/vox_fem/adios.ogg',

```

Comment:
Why

Reply:
Unnecessary, especially when we tout "use English here"